src/styles/blocks/card-grid.scss

```
@@ -0,0 +1,101 @@
+@import "../utils/variables";
+@import "../utils/functions";
+@import "../utils/mixins";
+@import "../utils/extends";
+
+.page__catalog {
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.product-grid {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  max-width: gridSize($grid-item-width, $grid-num-of-item,
+  $grid-margin-btw-item);
```

I believe, linter won't complain
```suggestion
  max-width: gridSize(
    $grid-item-width, 
    $grid-num-of-item,
    $grid-margin-btw-item
  );
```